configure_ironic.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -xe
+
+# Configure dnsmasq and pxe boot parameters.
+
+IRONIC_IP=${IRONIC_IP:-"172.22.0.1"}
+IRONIC_DHCP_RANGE=${IRONIC_DHCP_RANGE:-"172.22.0.10,172.22.0.100"}
+
```

Comment:
Would be useful to also set IRONIC_PORT to 5280 and use a variable in dnsmasq.conf and inspector.ipxe.

Reply:
I'd prefer to separate the port change completely from this one.